obspy/io/mseed/core.py

```diff
@@ -785,6 +785,16 @@ def _write_mseed(stream, filename, encoding=None, reclen=None, byteorder=None,
                 trace_attr['encoding'] = 1
             elif trace.data.dtype.type == np.dtype(native_str('|S1')).type:
                 trace_attr['encoding'] = 0
+            #int64 data not supported; if possible, downcast to int32, otherwise create error message
+            elif trace.data.dtype.type == np.int64:
```

Comment:
Looks like the recommended way to compare these is `....type is np.int64` (see https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.dtypes.html first example). Usually it's correct to use `==` but for singletons `is` is preferred usage in Python e.g. `something is None`

Reply:
Ah, nevermind, I just saw it's done like this in the whole file here. Just leave as is then to match the rest of the code.. ;-)